schema/attribute/bank-statement.json

```diff
@@ -0,0 +1,15 @@
+{
+	"$id": "http://platform.selfkey.org/schema/attribute/bank-statement.json",
+	"$schema": "http://platform.selfkey.org/schema/identity-attribute.json",
+	"idAttribute": true,
```

Comment:
Why do we need this value when we already have the `$schema` field? As long as this schema implements the `http://platform.selfkey.org/schema/identity-attribute.json` schema we know it's an identity attribute.

Reply:
Correct, will resolve

Reply:
@dfcowell Actually no, this is not enough. I can point the $schema to other compatible schemas, that inherit identity-attribute shema, same thing we did with the original json-schema

And we need another indicator that this json schema should be treated as identity attribute

Reply:
won't fix